pkg/log/log.go

```diff
@@ -0,0 +1,63 @@
+// Package log is a convenience wrapper for logging messages of various levels (associated colors to come)
+// to the terminal. Much of this code has been shamelessly stolen from https://github.com/helm/helm/blob/master/log/log.go
```

Comment:
lol... we need to write a decent implementation of this and put it in `deis/pkg`. I just nabbed the same piece of helm to mock up something else.